src/pseudopeople/interface.py

```diff
@@ -135,14 +168,14 @@ def _extract_columns(columns_to_keep, noised_dataset):
 def generate_decennial_census(
     source: Union[Path, str] = None,
     seed: int = 0,
-    config: Union[Path, str, dict] = None,
+    config: Union[Path, str, Dict[str, dict]] = None,
```

Comment:
Should the second `dict` be `Dict`? Same in every generate function

Reply:
It can be. My understanding is Dict and dict are the same except that typing.Dict allows for specifying key/value types. I'll change it for consistency